As upstream defaults create directories where binary is executed (/usr/bin/), this patch is needed to change paths to Gentoo specific locations
--- a/.env.template
+++ b/.env.template
@@ -5,10 +5,6 @@
 ## Be aware that most of these settings will be overridden if they were changed
 ## in the admin interface. Those overrides are stored within DATA_FOLDER/config.json .
 ##
-## By default, Vaultwarden expects for this file to be named ".env" and located
-## in the current working directory. If this is not the case, the environment
-## variable ENV_FILE can be set to the location of this file prior to starting
-## Vaultwarden.
 
 ####################
 ### Data folders ###
@@ -23,17 +19,17 @@
 ## When using an external location, make sure to set TMP_FOLDER,
 ## TEMPLATES_FOLDER, and DATABASE_URL to local paths and/or a remote database
 ## location.
-# DATA_FOLDER=data
+DATA_FOLDER=/var/lib/vaultwarden
 
 ## Individual folders, these override %DATA_FOLDER%
 # RSA_KEY_FILENAME=data/rsa_key
-# ICON_CACHE_FOLDER=data/icon_cache
+ICON_CACHE_FOLDER=/var/cache/vaultwarden/icon_cache
 # ATTACHMENTS_FOLDER=data/attachments
 # SENDS_FOLDER=data/sends
 
 ## Temporary folder used for storing temporary file uploads
 ## Must be a local path.
-# TMP_FOLDER=data/tmp
+TMP_FOLDER=/tmp/vaultwarden
 
 ## HTML template overrides data folder
 ## Must be a local path.
@@ -42,7 +38,7 @@
 # RELOAD_TEMPLATES=false
 
 ## Web vault settings
-# WEB_VAULT_FOLDER=web-vault/
+WEB_VAULT_FOLDER=/usr/share/webapps/vaultwarden-web
 # WEB_VAULT_ENABLED=true
 
 #########################
@@ -386,7 +382,7 @@
 # USE_SYSLOG=false
 
 ## Logging to file
-# LOG_FILE=/path/to/log
+LOG_FILE=/var/log/vaultwarden.log
 
 ## Log level
 ## Change the verbosity of the log output
